 2022-11-11 14:43:10,133  exeption occured
Traceback (most recent call last):
  File "d:\STUDENT PROJECT\school\student_api\errooo.py", line 9, in <module>
    c=a/b
ZeroDivisionError: division by zero
